deepfence_server/model/user.go

```diff
@@ -350,6 +350,21 @@ func GetUserByEmail(ctx context.Context, email string) (*User, int, *postgresqlD
 	return &user, http.StatusOK, pgClient, nil
 }
 
+func IsFreshSetup(ctx context.Context) (bool, error) {
+	pgClient, err := directory.PostgresClient(ctx)
+	if err != nil {
+		return false, err
+	}
+	_, err = pgClient.GetUsers(ctx)
```

Comment:
Recommend to use `pgClient.CountUsers(ctx)`

Reply:
pushed